docs/develop/protocol.md

```
@@ -44,23 +44,25 @@ If hami.io/node-handshake annotations remains in "Requesting_xxxx" and {schedule
-hami.io/{device-type}-devices-to-allocate:{ctr1 request}:{ctr2 request}:...{Last ctr request}:
+hami.io/vgpu-devices-to-allocate: {ctr1 request};{ctr2 request};...{Last ctr request};
 hami.io/vgpu-node: {schedule decision node}
 hami.io/vgpu-time: {timestamp}
 ```
 
-each container request is in the following format:
+`hami.io/vgpu-devices-to-allocate` is the key used for NVIDIA GPUs. Other device types use `hami.io/{device-type}-devices-to-allocate`, for example `hami.io/mthreads-vgpu-devices-to-allocate` or `hami.io/cambricon-mlu-devices-to-allocate`.
+
+each container request lists that container's devices. Every device is four comma-separated fields terminated by `:`, and every container is terminated by `;`:
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)